src/app.js

```diff
@@ -0,0 +1,22 @@
+import CardCreator from './create-card'
+import React from 'react'
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      cards: []
+    }
+    this.saveCard = this.saveCard.bind(this)
+  }
+  saveCard(card) {
+    this.state.cards.push(card)
```

Comment:
Be sure not to mutate `state` or its properties directly. Make a copy and use `setState` instead.